salt/runners/venafiapi.py

```diff
@@ -41,7 +43,7 @@
 from salt.exceptions import CommandExecutionError
 
 __virtualname__ = 'venafi'
-base_url = 'https://api.beta.venafi.com/v1'
+base_url = 'http://vpc-51255c36.qa.projectc.venafi.com/v1'
```

Comment:
Is there a way to set the URL for the service as a parameter in the master file? Otherwise, we should remove this URL and put in a placeholder with instructions for the user to put in the actual URL before running the module.